src/styles/focusMode.css

```
@@ -0,0 +1,31 @@
+/* Hide recommendations sidebar on both watch and playlist pages */
+body.tmc-focus-mode ytd-watch-next-secondary-results-renderer {
+    display: none;
+}
```

```suggestion
/* Hide recommendations sidebar on watch pages */
body.tmc-focus-mode #related {
    display: none;
}
```

`#related` is a better selector here.